language-support/codegen-common/src/main/scala/com/digitalasset/daml/lf/codegen/conf/Conf.scala

```diff
@@ -57,6 +58,11 @@ object Conf {
       .action((l, c) => c.copy(verbosity = l))
       .text("Verbosity between 0 (only show errors) and 4 (show all messages) -- defaults to 0")
 
+    opt[String]('r', "root")(Read.stringRead)
```

Comment:
this change is also going to affect java codegen... and the SDK doc, that does not mention `root`.

Reply:
@leo-da We can remove the option from the driver if you prefer but I do think it is useful to have in all codegen.

Reply:
@jberthold-da that is fine, I was just pointing the things that we will need to address when this PR lands in master.

Reply:
@leo-da so should I take the option out of the CLI/config, or just declare it as unsupported for java?
I do think it is a very useful option.